scripts/xe-toolstack-restart

```diff
@@ -30,7 +30,7 @@ if [ $POOLCONF == "master" ]; then MPATHALERT="mpathalert"; else MPATHALERT="";
 SERVICES="perfmon v6d xenopsd xenopsd-xc xenopsd-xenlight
   xenopsd-simulator xenopsd-libvirt xcp-rrdd-iostat xcp-rrdd-squeezed
   xcp-rrdd-xenpm xcp-rrdd-gpumon xcp-rrdd xcp-networkd squeezed forkexecd
-  $MPATHALERT xapi-storage-script xapi-clusterd varstored-guard message-switch"
+  $MPATHALERT xapi-storage-script xapi-clusterd message-switch varstored-guard"
```

Comment:
Can we put message-switch to the very beginning? Does message-switch have any dependencies of its own?

Reply:
I expect all the toolstack daemons to depend on it, so it should be shut down last

Reply:
Yes, let's make that change, and target the master branch.